src/linear_algebra/affine_space/basic.lean

```diff
@@ -26,15 +26,8 @@ files:
 * `affine_subspace`: a subset of an affine space closed w.r.t. affine combinations of points;
 * `affine_combination`: an affine combination of points;
 * `affine_independent`: affine independent set of points;
-* `barycentric_coord`: the barycentric coordinate of a point.
+* `affine_basis.coord`: the barycentric coordinate of a point.
 
-## TODO
-
-Some key definitions are not yet present.
-
-* Affine frames.  An affine frame might perhaps be represented as an `affine_equiv` to a `finsupp`
-  (in the general case) or function type (in the finite-dimensional case) that gives the
-  coordinates, with appropriate proofs of existence when `k` is a field.
```

Comment:
This removed remark is about affine frames (origin plus basis for the underlying module), not about barycentric frames (affine independent spanning points), so I don't think removing it fits in with the rest of this PR.

Reply:
Agreed!

Thank you for the clarifying remarks. I have restored your comment, which I had not properly understood.